src/intervals/rounding.jl

```diff
@@ -9,6 +9,7 @@ interval rounding types, e.g.
 +(IntervalRounding{:none}, a, b, RoundDown)
 
 The current allowed rounding types are
+- :errorfree # use errorfree arithmetic via FastRounding.jl
```

Comment:
It should also be mentioned somewhere (the docstrings and perhaps a warning too) that `:errorfree` works only with `Float64` and `Float32`.